store/mockstore/unistore/cophandler/cop_handler.go

```diff
@@ -84,8 +84,8 @@ type dagContext struct {
 	startTS       uint64
 }
 
-// handleCopDAGRequest handles coprocessor DAG request.
-func handleCopDAGRequest(dbReader *dbreader.DBReader, lockStore *lockstore.MemStore, req *coprocessor.Request) (resp *coprocessor.Response) {
+// handleCopDAGRequestWithMPP handles coprocessor DAG request using MPP executors.
+func handleCopDAGRequestWithMPP(dbReader *dbreader.DBReader, lockStore *lockstore.MemStore, req *coprocessor.Request) (resp *coprocessor.Response) {
```

Comment:
Looks like there is only one test framework to handle both mpp/non-mpp dag request, if it is true, I think there is no need to emphasize mpp, just use `handleCopDAGRequest` is ok